Classification/cnns/train.sh

```diff
@@ -18,48 +19,52 @@ python3 of_cnn_train_val.py \
 
 
 
+
 # # training with mini-imagenet
 # DATA_ROOT=data/mini-imagenet/ofrecord
 # python3 of_cnn_train_val.py \
-#     --train_data_dir=$DATA_ROOT/train \
-#     --num_examples=50 \
-#     --train_data_part_num=1 \
-#     --val_data_dir=$DATA_ROOT/validation \
-#     --num_val_examples=50 \
-#     --val_data_part_num=1 \
-#     --num_nodes=1 \
-#     --gpu_num_per_node=1 \
-#     --model_update="momentum" \
-#     --learning_rate=0.001 \
-#     --loss_print_every_n_iter=1 \
-#     --batch_size_per_device=16 \
-#     --val_batch_size_per_device=10 \
-#     --num_epoch=10 \
-#     --model="resnet50"
+#    --train_data_dir=$DATA_ROOT/train \
+#    --num_examples=50 \
+#    --train_data_part_num=1 \
+#    --val_data_dir=$DATA_ROOT/validation \
+#    --num_val_examples=50 \
+#    --val_data_part_num=1 \
+#    --num_nodes=1 \
+#    --gpu_num_per_node=1 \
+#    --optimizer="sgd" \
+#    --momentum=0.875 \
+#    --learning_rate=0.001 \
+#    --loss_print_every_n_iter=1 \
+#    --batch_size_per_device=16 \
+#    --val_batch_size_per_device=10 \
+#    --num_epoch=10 \
+#    --model="resnet50"
```

Comment:
changes above are not necessary

Reply:
We can leave these comments unchanged in this PR.
or start a new PR to remove useless comment lines like here or below.

Reply:
> changes above are not necessary


I think its necessary and friendly to use ,considering we have three ways to training model:
1.training with synthetic data
2.training with mini-imagenet
2.training with imagenet

Reply:
These are commented lines, I suggest to keep them unchanged, and start a new PR to handle them.
In the new PR, we can remove all commented lines, and update README about these three options.

Reply:
ok,agree